rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtils.java

```diff
@@ -121,4 +131,17 @@ private void fillPatientData(JSONObject field, String prefix, String value) thro
             dobObject.put(JsonFormConstants.TEXT, prefix + value);
         }
     }
+
+    private void prePopulateSampleShipmentDetailsFormFields(JSONObject field) throws JSONException {
+
+        // pre-populate the sender name
+        if (CovidConstants.FormFields.SENDER_NAME.equals(field.getString(KEY))) {
+            field.put(JsonFormConstants.VALUE, getNameOfSampler());
+        }
+    }
+
+    private String getNameOfSampler() {
```

Comment:
Please rename this to `getLoggedInUserPrefferredName`.

Reply:
fixed.